README.md

```diff
@@ -85,15 +86,52 @@ During a successful onboarding process, a new device will be created in NetBox w
 
 > By default, the plugin is using the credentials defined in the main `configuration.py` for Napalm (`NAPALM_USERNAME`/`NAPALM_PASSWORD`). It's possible to define specific credentials for each onboarding task.
 
+## Stack Platforms
+
+Onboarding plugin now supports the onboarding of specifically tested platforms. These will be added to NetBox as
+individual devices. The primary (first) unit in the stack will:
+
+- Have the primary IP address associated (if configured, defaults to yes)
+- Will **not** have the `stack_separator` in the name of the device, by using colon (:) this would
+override any default application port with automation systems.
+
+All of the additional units will have the separator_indicator separating the name from the unit
+number. There **will not** be a primary IP address assigned to the unit. Assuming that there is a
+methodology to parse each of the model numbers and serial numbers from a command output, there will
+be added in about the device that is unique to the device of serial number, model number.
```

Comment:
Last sentence here is a bit confusing to me. Would the following be a correct restatement?

"If this plugin implements a methodology to parse per-unit information (such as serial number, model number, etc.) from a command output, then this information will be added to the unit."

Reply:
I agree. It made sense in the time of writing.